cime_config/tests.py

```diff
@@ -436,16 +435,15 @@
 
     "e3sm_gpuomp" : {
         "tests"    : (
-                 "SMS_Ln9.ne4pg2_ne4pg2.F-MMFOMP.eam-single_thread",
                  "SMS_Ld1.T62_oEC60to30v3.DTESTM",
                  )
     },
 
     "e3sm_gpucxx" : {
         "tests"    : (
-                 "SMS_Ln9.ne4pg2_ne4pg2.F-MMFXX.eam-mmf_fixed_subcycle",
-                 "SMS_Ln9.ne4pg2_oQU480.F20TR-MMFXX.eam-mmf_use_VT",
-                 "SMS_Ln9.ne4pg2_ne4pg2.F-MMFXX.eam-mmf_use_ESMT",
+                 "SMS_Ln9.ne4pg2_ne4pg2.F2010-MMF1.eam-mmf_fixed_subcycle",
+                 "SMS_Ln9.ne4pg2_oQU480.F20TR-MMF1.eam-mmf_use_VT",
```

Comment:
same thing over here. you could put line 446 ("SMS_Ln9.ne4pg2_ne4pg2.F2010-MMF1.eam-mmf_use_ESMT") right after line 444 just so that compsets align better.